src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -559,6 +562,13 @@ def __init__(self, raw_response):
             fileTypeSummaries=[FileTypeSummary.for_bucket(bucket) for bucket in _sum['buckets']],
             cellCountSummaries=[OrganCellCountSummary.for_bucket(bucket) for bucket in _organ_group['buckets']])
 
+        # Un-translate values that had been translated with translate_fields() to handle Nones in Elasticsearch
```

Comment:
Check out what I did here in the next commit. It's odd that a list is generated with a comprehension and then patched in place 5 lines down. My changes will reduce the churn in state and be more efficient.